releasenotes/notes/27696.yaml

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
I'm actually not sure how this should work -- since this is a cherrypick, do I need to change the release note file to point to the cherrypick issue/pr?